README.md

```diff
@@ -36,10 +51,15 @@ sudo apt-get install python3-pip
 ### Yaml-cpp
 
 If this software package is not available in the system (it is shipped with some software, e.g. ROS), dowload it and install it from <https://github.com/jbeder/yaml-cpp>. **Please make sure it is installed in a share library mode**.
+Klepsydra [SDK tutorial](https://github.com/klepsydra-technologies/kpsr-tutorial/blob/main/README.md) will require yaml support.
```

Comment:
No need to add this line here because kpsr-tutorial already mentions that YAML support is needed.

Reply:
Just to prevent building twice once you came to the tutorial.